lte/gateway/c/core/oai/include/state_converter.h

```diff
@@ -168,7 +168,7 @@ class StateConverter {
 
  private:
   static void plmn_to_chars(const plmn_t& state_plmn, char* plmn_array);
-  static void chars_to_plmn(const char* plmn_array, plmn_t& state_plmn);
+  static void chars_to_plmn(const char* plmn_array, plmn_t* state_plmn);
```

Comment:
what is wrong with pass by reference?

Reply:
Cpp lint complains if you use reference for non-const references.

Reply:
strange. okay let's go with linter then...